composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "magento-hackathon/e-mail-preview",
+    "name": "schnumsel/e-mail-preview",
```

Comment:
Of course, your right. I did this, not knowing, i could define another branch to fetch the schnumsel stuff via composer.